e2e/test/scenarios/dashboard-filters/shared/dashboard-filters-text-category.js

```diff
@@ -1,26 +1,67 @@
-export const DASHBOARD_TEXT_FILTERS = {
-  Is: {
+export const DASHBOARD_TEXT_FILTERS = [
+  {
+    operator: "Is",
+    single: true,
     value: "Organic",
     representativeResult: "39.58",
   },
-  "Is not": {
+  {
+    operator: "Is not",
+    single: true,
     value: "Organic",
     representativeResult: "37.65",
   },
-  Contains: {
-    value: "oo",
+  // It is important to keep multiple values as a single string in the value field.
+  {
+    operator: "Contains",
+    value: "oo,aa",
     representativeResult: "148.23",
+    negativeAssertion: "37.65",
   },
-  "Does not contain": {
-    value: "oo",
+  {
+    operator: "Contains",
+    single: true,
+    value: "oo,aa",
+    representativeResult: "No results!",
+    negativeAssertion: "148.23",
+  },
+  {
+    operator: "Does not contain",
+    value: "oo,tt",
+    representativeResult: "39.58",
+    negativeASsertion: "37.65",
```

Comment:
There are more occurrences of this typo in this file. Converting it to TS should not be too difficult.

```suggestion
    negativeAssertion: "37.65",
```